### PR TITLE
Feature hcw 306 pii minors

### PR DESCRIPTION
**Pull request by DevChima**

### CODE REVIEW

File: base/actions/actions.py [hunk 1]
```diff
@@ -584,10 +625,14 @@ def get_healthcheck_data(self, tracker: Tracker, risk: Text) -> Dict[Text, Any]:
         }
 
         location = self.fix_location_format(tracker.get_slot("location_coords"))
+        city_location = self.fix_location_format(
+            tracker.get_slot("city_location_coords")
+        )
```
Comment (erikh360) [inline base/actions/actions.py]: Should this maybe move into the if below, then we can simplify it quite a bit.

I would think its safe to assume `*_coords` would have values if age != <18

**Review by erikh360 — COMMENTED** [review]
Just the one comment on the code + I think a test or two for `get_healthcheck_data` would be cool. Just to make sure it builds the data dict correctly.

**Comment by erikh360** [comment]
I see the tests I suggested are already there.

File: base/actions/actions.py (hunk 1 above)
Comment (DevChima) [reply base/actions/actions.py]: I've noticed from several attempts that you do not always get to see the location entry option especially right after a previous submission. Sometimes I have to wait a few minutes to send again before I see all the options.

Comment (erikh360) [reply base/actions/actions.py]: My comment is more about simplifying the code

**Pull request merged**